### DISCUSSION TITLE
No email notification for users created in MediaCMS Administration

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#122] [Q&A] [answered]
When I create a user from the MediaCMS Administration, I as site admin receive an email that a new user just registered, but the new user doesn’t receive an email at their address that they have a new account. Since I am not setting their password, I assume the user must receive an email with their password or a link to set their password.
How is this intended to work?

**Comment by mgogoulos** [comment, marked as answer]
When self registration is enabled, users create their own accounts and set password. In this case site admins receive an email that a new user has registered, users don't receive an email but we could setup a welcome email for them as well (apparently this should be on a place where site admins can easily edit). 

But what you describe here is a different case, I'm assuming self registration is disabled, so you create accounts as admin through the Django dashboard (MediaCMS Administration), this is a scenario I haven't checked. Since MediaCMS relies on https://django-allauth.readthedocs.io for user handling, perhaps there's an option already there for this case (django-all-auth is very powerfhul!)

**Reply by Dan1ell** [reply]
I have self-registration enabled. That works fine. 
I was just making some accounts for myself in the admin and realized I could not get to the passwords for them. I don't enter a password when making them in the admin and I'm not emailed the auto-generated passwords either. So how do l log in as those users? I don't know their passwords.

**Reply by Dan1ell** [reply]
I have created an issue for this: #139
